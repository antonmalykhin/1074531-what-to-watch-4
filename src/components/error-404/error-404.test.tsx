import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
import history from '../../history';
import Error404 from './error-404';

it(`Error page render correctly`, () => {
  const tree = renderer
    .create(
        <Router history={history}>
          <Error404 />
        </Router>
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
