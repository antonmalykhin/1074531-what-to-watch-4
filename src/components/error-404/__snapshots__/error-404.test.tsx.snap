// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page render correctly 1`] = `
<div
  className="error"
>
  <p
    className="error__title"
  >
    404
  </p>
  <p
    className="error__text"
  >
    Page is not found
  </p>
  <a
    className="error__link"
    href="/"
    onClick={[Function]}
  >
    Back to the main page
  </a>
</div>
`;
